### ISSUE TITLE
Math.sign is not available in IE

### ISSUE THREAD

**Issue by manoharprabhu** [mathisonian/premonish#2]
<img width="617" alt="mathsign" src="https://cloud.githubusercontent.com/assets/2051859/20877943/ee5ef19a-baf1-11e6-8caf-f8526292e01c.PNG">

<hr />

A Polyfill can probably be used, if not available.
[Math.sign](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/sign)

**Comment by Qix-** [comment]
They have a polyfill for it on MDN already.

```javascript
Math.sign = Math.sign || function(x) {
  x = +x; // convert to a number
  if (x === 0 || isNaN(x)) {
    return Number(x);
  }
  return x > 0 ? 1 : -1;
}
```

**Comment by mathisonian** [comment]
fixed by #3
